lib/elixir/test/elixir/fixtures/diagnostics/unicode_error._ex

```diff
@@ -0,0 +1,5 @@
+defmodule Sample do
+  def a do
+    10 + 😎
+  end
+end
```

Comment:
Let's move this to the test file itself please. :) Either directly in the function or as a module attribute if used in multiple places.